compiler-maven-plugin/src/main/java/com/dylibso/chicory/build/time/maven/ChicoryCompilerGenMojo.java

```diff
@@ -17,8 +17,11 @@
 /**
  * This plugin generates an invokable library from the compiled Wasm
  */
-@Mojo(name = "wasm-aot-gen", defaultPhase = LifecyclePhase.GENERATE_SOURCES, threadSafe = true)
-public class AotGenMojo extends AbstractMojo {
+@Mojo(
+        name = "chicory-compiler-gen",
```

Comment:
mvn plugins already have the group id as namespace.  So I think this should be something simpler like `name = "compile"`

Reply:
I disagreee.  it's always define in the scope of the plugin so it's obvious your doing a chicory compile.

```xml
    <plugin>
      <groupId>com.dylibso.chicory</groupId>
      <artifactId>chicory-compiler-maven-plugin</artifactId>
      <executions>
        <execution>
          <id>compile</id>
          <goals>
            <goal>compile</goal>
          </goals>
        </execution>
      </executions>
    </plugin>
```

Reply:
Ok, integrating the change in an iteration on #922